### PR TITLE
New animation for slide without padding and margin

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-SteveBrush I agree but couldn't think of anything better :)

File: src/app/public/slide.ts [hunk 1]
```diff
@@ -26,3 +26,26 @@ export const skyAnimationSlide = trigger('skyAnimationSlide', [
     animate('150ms ease-in')
   )
 ]) as AnimationEntryMetadata;
+
+export const skyAnimationSlideNoTrim = trigger('skyAnimationSlideNoTrim', [
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/slide.ts]: @Blackbaud-TrevorBurch would you mind adding some documentation here to denote what makes this one unique over its predecessor?

**Comment by Blackbaud-AlexKingman** [comment]
Last question: I noticed this only affects top & bottom trim. Will we ever need to do left & right? Should we call this `skyAnimationSlideNoVerticalTrim`?

**Comment by Blackbaud-TrevorBurch** [comment]
Talked to Alex and we decided not to change the name as the other vertical slide doesn't call it out.

**Comment by Blackbaud-TrevorBurch** [comment]
Closing as we worked around needing this.

**Pull request closed without merging**